src/brasil/gov/temas/jbot/Products.TinyMCE.skins.tinymce.plugins.plonebrowser.plonebrowser.htm.pt

```diff
@@ -37,7 +37,7 @@
                 </div>
                 <div id="tippreview_link">
                     <a href="#tippreview">
-                      <img src="++resource++portalbrasil/img/preview.png" />
+                      <img src="++theme++padrao/img/preview.png" />
```

Comment:
the icon is the same for both themes, so there is no problem to keep this fixed to theme padrao